utils/gcp.py

```diff
@@ -406,6 +406,51 @@ def bigquery_dq_check(
         **kwargs,
     )
 
+def bigquery_bigeye_check(
+    task_id,
+    table_id,
+    warehouse_id,
+    project_id="moz-fx-data-shared-prod",
+    gcp_conn_id="google_cloud_airflow_gke",
+    gke_project_id=GCP_PROJECT_ID,
+    gke_location="us-west1",
+    gke_cluster_name="workloads-prod-v1",
+    gke_namespace="default",
+    docker_image="gcr.io/moz-fx-data-airflow-prod-88e0/bigquery-etl:latest",
```

Comment:
I assume this image is now referenced in multiple places and should always be the same. Maybe it would be worth making this a constant value on top of the module and referencing that everywhere instead to avoid accidental use of different bqetl images?